talk/tools.tex

```diff
@@ -683,3 +683,49 @@
     \end{itemize}
   \end{alertblock}
 \end{frame}
+
+\begin{frame}[fragile]
+  \frametitle{clang-tidy}
+  \begin{block}{compile\_commands.json}
+    \begin{itemize}
+      \item clang-tidy needs to know exactly how your program is built
+      \item tries to read this from a file called compile\_commands.json
+      \item generate with CMake: \\
+        \mintinline{bash}{cmake -DCMAKE_EXPORT_COMPILE_COMMANDS=ON ...}
+      \item for Makefiles try \href{https://github.com/rizsotto/Bear}{\beamergotobutton{Bear}}
+    \end{itemize}
+  \end{block}
+  \begin{block}{.clang-tidy}
+    \begin{itemize}
+      \item file describing which checks to run
+    \end{itemize}
+  \end{block}
+  \begin{block}{Supported checks}
+    \begin{itemize}
+      \item \url{https://clang.llvm.org/extra/clang-tidy/checks/list.html}
+    \end{itemize}
+  \end{block}
+\end{frame}
+
+\begin{frame}[fragile]
+  \frametitle{clang-tidy}
+  \begin{block}{Run clang-tidy}
+    \begin{itemize}
+      \item \mintinline{bash}{clang-tidy <file.cpp> -checks=...}
+      \item \mintinline{bash}{clang-tidy <file.cpp>} (checks from .clang-tidy file)
+      \item \mintinline{bash}{clang-tidy <file.cpp> --fix} (applies fixes automatically)
```

Comment:
This goes on 2 lines and then the slide does not fit anymore. I would simply remove the word "automatically"
```suggestion
      \item \mintinline{bash}{clang-tidy <file.cpp> --fix} (applies fixes)
```

Reply:
thx